.ci/release_template.md

```diff
@@ -57,16 +57,13 @@ Use these label badges for highlighting important and key changes:
  -->
 
  - ⚠️ **Important:**
-    - --REPLACE-WITH-HIGHLIGHTED-IMPORTANT-CHANGES--
 
  - ✨ **New Features:**
-    - --REPLACE-WITH-HIGHLIGHTED-NEW-FEATURES--
 
  - 🐛 **Fixed Bugs / Resolved issues:**
-    - --REPLACE-WITH-HIGHLIGHTED-FIXED-BUGS--
 
 <details>
-<summary>📘 <b>Show all changes <code> --REPLACE-WITH-CHANGES-COUNT-- </code></b></summary>
+<summary>📘 <b>Show all changes <code> xx </code></b></summary>
```

Comment:
comments use this notation `<!-- comment -->`

Reply:
I did not intent to do so, but sure.

Reply:
Then how else would we know what xx means?